### ISSUE TITLE
Update resource (URLs) browser

### ISSUE THREAD

**Issue by SuaYoo** [webrecorder/replayweb.page#241]
### Changes
- Show number of resources
- Group by timestamp
- Migrate to shoelace

### Mockups

**Comment by SuaYoo** [comment]
@Shrinks99

**Comment by Shrinks99** [comment]
Yes!  I believe the pages list only has single entries per URL, but the resources list still has one for each timestamp.  This can be a little hard to find in a given archive because the resources list can't be searched in any meaningful way until everything is loaded in with scrolling, and then it's immediately loaded out once you search... but that's a separate issue.
